src/utils/util_load.cpp

```diff
@@ -932,6 +933,11 @@ int main(int argc, char **argv){
   opt["value"].append(weight_bonus);
   conf.addOption("extra", opt);
 
+  opt["short"] = "S";
+  opt["long"] = "usedtscs";
```

Comment:
Nit: my brain parsed this as "used tscs". Perhaps `--secure` gets the point across a little clearer?

Reply:
Yep, will change it to your suggestion.